lists/it.csv

```diff
@@ -0,0 +1,333 @@
+http://rarbg.to,FILE,File-sharing,2017-04-12,,Site reported to be blocked by AGCOM - Italian Autority on Communication
+http://giochitalianids.com,FILE,File-sharing,2017-04-12,,Site reported to be blocked by AGCOM - Italian Autority on Communication
+http://games.torrentsnack.com,FILE,File-sharing,2017-04-12,,Site reported to be blocked by AGCOM - Italian Autority on Communication
+http://mega-wii.com,FILE,File-sharing,2017-04-12,,Site reported to be blocked by AGCOM - Italian Autority on Communication
+http://nowdownload.ec,FILE,File-sharing,2017-04-12,,Site reported to be blocked by AGCOM - Italian Autority on Communication
+http://nowdownload.to,FILE,File-sharing,2017-04-12,,Site reported to be blocked by AGCOM - Italian Autority on Communication
+http://altadefinizione01.cool,FILE,File-sharing,2017-04-12,,Site reported to be blocked by AGCOM - Italian Autority on Communication
+http://newalbumreleases.net,FILE,File-sharing,2017-04-12,,Site reported to be blocked by AGCOM - Italian Autority on Communication
+http://instamp3.co,FILE,File-sharing,2017-04-12,,Site reported to be blocked by AGCOM - Italian Autority on Communication
+http://123movies.is,FILE,File-sharing,2017-04-12,,Site reported to be blocked by AGCOM - Italian Autority on Communication
+http://123movies.to,FILE,File-sharing,2017-04-12,,Site reported to be blocked by AGCOM - Italian Autority on Communication
+http://www.lastampa.it,NEWS,News Media,2017-04-12,,
+http://www.vanityfair.it,NEWS,News Media,2017-04-12,,
+http://www.repubblica.it,NEWS,News Media,2017-04-12,,
+http://www.corriere.it,NEWS,News Media,2017-04-12,,
+http://www.ilsole24ore.com,NEWS,News Media,2017-04-12,,
+http://www.ilmessaggero.it,NEWS,News Media,2017-04-12,,
+http://www.ilrestodelcarlino.it,NEWS,News Media,2017-04-12,,
+http://www.avvenire.it,NEWS,News Media,2017-04-12,,
+http://www.lanazione.it,NEWS,News Media,2017-04-12,,
+http://www.liberoquotidiano.it,NEWS,News Media,2017-04-12,,
+http://www.ilgiorno.it,NEWS,News Media,2017-04-12,,
+http://www.ilmattino.it,NEWS,News Media,2017-04-12,,
+http://www.italiaoggi.it,NEWS,News Media,2017-04-12,,
+http://www.ilfattoquotidiano.it,NEWS,News Media,2017-04-12,,
+http://www.ilmanifesto.info,NEWS,News Media,2017-04-12,,
+http://www.linkiesta.it,NEWS,News Media,2017-04-12,,
+http://www.valigiablu.it,NEWS,News Media,2017-04-12,,
+http://www.espresso.repubblica.it,NEWS,News Media,2017-04-12,,
+http://www.panorama.it,NEWS,News Media,2017-04-12,,
+http://www.agi.it,NEWS,News Media,2017-04-12,,
+http://www.ansa.it,NEWS,News Media,2017-04-12,,
+http://www.quotidiano.net,NEWS,News Media,2017-04-12,,
+http://www.milanofinanza.it,NEWS,News Media,2017-04-12,,
+http://www.leggo.it,NEWS,News Media,2017-04-12,,
+http://city.corriere.it,NEWS,News Media,2017-04-12,,
+http://www.dnews.eu,NEWS,News Media,2017-04-12,,
+http://www.tgcom24.mediaset.it,NEWS,News Media,2017-04-12,,
+http://www.wired.it,NEWS,News Media,2017-04-12,,
+http://www.vice.com/it,NEWS,News Media,2017-04-12,,
```

Comment:
Could you add the source column as well? It can be anything including anonymous if you don't want to be credited or some moniker of your choosing.